public/css/icinga/menu.less

```diff
@@ -271,27 +273,29 @@
     margin-top: -1px; // Align content with the menu item, not its border
 
     &::after {
+      --caret-side: calc(~"1em + 2px");
+
       .transform(rotate(45deg));
 
       background-color: @body-bg-color;
       border-bottom: 1px solid @gray-light;
       border-left: 1px solid @gray-light;
       content: "";
       display: block;
-      height: 1.1em;
-      width: 1.1em;
+      height: var(--caret-side);
+      width: var(--caret-side);
       position: absolute;
-      top: ~"calc((@{nav-item-height} / 2) - (1.1em / 2))";
```

Comment:
Huh? Just replace `1.1em` with `var(--caret-side)` please, like you did in bottom-up's ::after.